docs/api-v2-import.json

```diff
@@ -0,0 +1,520 @@
+{
+    "openapi": "3.0.0",
+    "info": {
+      "version": "2.0.0",
+      "title": "Scarf data import api",
+      "description": "This API specification is for importing event data into Scarf"
+    },
+    "servers": [
+      {
+        "url": "http://api.scarf.sh"
+      }
+    ],
+    "paths": {
+      "/v2/imports/{owner}/{ticket}/log": {
+        "parameters": [
+          {
+            "name": "owner",
+            "in": "path",
+            "required": true,
+            "schema": {
+              "type": "string"
+            },
+            "description": "Owner of the event import.\n"
+          },
+          {
+            "name": "ticket",
+            "in": "path",
+            "required": true,
+            "schema": {
+              "type": "string"
+            },
+            "description": "Id of the event import to retrieve logs for.\n"
+          }
+        ],
+        "get": {
+          "summary": "Stream the import log for a specified event import",
```

Comment:
What is an import log? Let's explain what that is, why you'd want to do it, etc.